ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/ActionManager.java

```diff
@@ -41,7 +42,7 @@ public long getTimeAdded ()
 		private long timeAdded;
 	}
 	
-	public static final ActionManager manager()
+	public static ActionManager manager()
```

Comment:
Is it necessary to no longer mark it as final?

Reply:
`final` is a logical error here (although the compiler allows it). I will add this modification to the new JBTM as well

Reply:
Done. New [JBTM](https://issues.redhat.com/browse/JBTM-3897) created. Thanks Tom